lib/src/scss-shared/_nx-input-helpers.scss

```diff
@@ -32,6 +35,13 @@
           stroke: #02A6D9;
         }
       }
+      &:hover {
+        + .nx-radio__circle {
```

Comment:
This shouldn't be in the mixin, it is specific to the radio.

Reply:
> This shouldn't be in the mixin, it is specific to the radio.

Everything within this portion of the mixin (under `&:focus-within {`) is specific to the radio.

Reply:
> Everything within this portion of the mixin (under &:focus-within {) is specific to the radio.

That isn't exactly true, though I can see that the selector immediately above this one is specific to the radio. I guess that makes me also question why that selector is in the mixin.